src/components/text/Headline.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import {HEADLINE_COLOR, HEADLINE_SIZE, HEADLINE_TYPE, HEADLINE_TRANSFORM} from './headlineConsts';
+import {HEADLINE_COLOR, HEADLINE_SIZE, HEADLINE_TYPE, HEADLINE_TRANSFORM, HEADLINE_ALIGN} from './headlineConsts';
 
-export {HEADLINE_TYPE, HEADLINE_SIZE, HEADLINE_COLOR, HEADLINE_TRANSFORM} from './headlineConsts';
+export {HEADLINE_TYPE, HEADLINE_SIZE, HEADLINE_COLOR, HEADLINE_TRANSFORM, HEADLINE_ALIGN} from './headlineConsts';
 
 const Headline = ({
   children,
   type = HEADLINE_TYPE.H1,
   size = HEADLINE_SIZE.NORMAL,
   extraBold,
   transform,
+  align,
```

Comment:
@burakukula it's a verb, I think we should use a noun here: `alignment`, wdyt?

Reply:
or there's a reason to do it this way?

Reply:
@aju @maciekbb wdyt?

Reply:
@vergilius hmm, I thought more like: align-left/align-right, if we change it we should probably change it everywhere - we already use this pattern in SG.

Reply:
@vergilius acctualy, we have both patterns now. 🤯

Reply:
@burakukula okay, sounds reasonable

Reply:
@aju @maciekbb @vergilius I assume, `ALIGNMENT` in context of Content box and `TEXT_ALIGN`/`HEADLINE_ALIGN` in context of text make sense? or do you think we should have the same pattern everywhere?